packages/client/integration/Loc.ts

```diff
@@ -376,16 +399,24 @@ export async function identityLoc(state: State) {
             country: "Wonderland"
         },
         draft: false,
-    });
+    }) as PendingRequest;
 
     locsState = pendingRequest.locsState();
-    expect(locsState.pendingRequests["Identity"][0].data().status).toBe("REQUESTED");
+    expect(locsState.pendingRequests["Identity"][0].data().status).toBe("REVIEW_PENDING");
+
+    await legalOfficer.acceptLoc(pendingRequest.locId);
```

Comment:
Replace by a call to new LO API?

Reply:
Good suggestion. This will be addressed later on in logion-network/logion-internal#923